scripts/replication.sh

```diff
@@ -1,13 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Are these steps documented anywhere?

This strategy is very handy to bootstrap the application without having to manually run a full import against the CKAN API.